po/ru.po

```diff
@@ -3620,79 +3620,82 @@ msgstr "Открыть в браузере?"
 #: app/flatpak-cli-transaction.c:683
 #, c-format
 msgid "Login required remote %s (realm %s)\n"
-msgstr "Удаленному репозиторию %s необходим логин (область %s)\n"
+msgstr "Удалённому репозиторию %s необходим логин (область %s)\n"
 
 #: app/flatpak-cli-transaction.c:688
 msgid "Password"
 msgstr "Пароль"
 
 #. Only runtimes can be pinned
 #: app/flatpak-cli-transaction.c:743
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
 "Info: (pinned) runtime %s%s%s branch %s%s%s is end-of-life, in favor of "
 "%s%s%s branch %s%s%s\n"
 msgstr ""
-"Информация: конец срока поддержки для закреплённого %s//%s наступил в пользу "
-"%s\n"
+"\n"
+"Информация: конец срока поддержки среды выполнения %s%s%s ветви %s%s%s в пользу "
+"%s%s%s ветви %s%s%s\n"
 
 #: app/flatpak-cli-transaction.c:749
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "\n"
 "Info: runtime %s%s%s branch %s%s%s is end-of-life, in favor of %s%s%s branch "
 "%s%s%s\n"
 msgstr ""
 "\n"
-"Информация: %s%s %s прекращает свое существование, в пользу %s\n"
+"Информация: среда выполнения %s%s%s ветви %s%s% прекращает свое существование в пользу %s%s%s ветви "
```

Comment:
```suggestion
"Информация: среда выполнения %s%s%s ветви %s%s%s прекращает свое существование в пользу %s%s%s ветви "
```

Reply:
Will you submit one change at a time? :)

Reply:
> Will you submit one change at a time? :)

Not to confuse yourself :) It's done.